components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -396,6 +408,16 @@ export class ParsedCommand {
                         id: 'apps.error.parser.unexpected_whitespace',
                         defaultMessage: 'Unreachable: Unexpected whitespace.',
                     }));
+                case '[':
+                    if (!this.field?.multiselect) {
+                        return this.asError(this.intl.formatMessage({
+                            id: 'apps.error.parser.unexpected_squared_bracket',
+                            defaultMessage: 'Unexpected list opening.',
+                        }));
+                    }
```

Comment:
What if it makes sense in the user's case for a non-multiselect text value to begin with `[`?

Reply:
They would have to use quotes. Not sure if we can make it obvious.